README.md

```diff
@@ -1,277 +1,1037 @@
+incentives for users to provide liquidity with two-way swaps and arbitragers to compose complex
+swaps. As long as the entry and exit swap pairs have enough liquidity, arbitragers can spread
+liquidity into less liquid swap pairs. As a bonus, **the very nature of *illiquidity* implies great
+arbitrage opportunities**. The more illiquid a swap pair, the greater the potential arbitrage
+profits. Providing liquidity and participating in arbitrage is permissionless, so anyone can create
+their own strategies/algorithms to provide liquidity or find the most profitable "path" through the
+sea of available swaps.
 
-Given the performance of these Aiken contracts, Cardano-Swaps is more than performant enough for the current state of Cardano. **No CIPs or hard-forks are needed. This protocol works on the Cardano blockchain, as is.**
+Since stake pool operators are required to always be connected to the network, they are uniquely
+positioned to serve as arbitragers. It can provide another potential source of profit for all stake
+pools operators, including the small pool operator who rarely makes blocks.
 
-Full benchmarking details can be found in the [Benchmarks](Benchmarks.md) file.
+To fully appreciate the implications of the two-way swaps, it helps to consider a few examples:
 
+##### The Contrived Arbitrage Example
 
-## Future Directions
-Features and considerations for future versions of Cardano-Swaps:
+``` Txt
+Alice has 10 ada in her swap address and is willing to swap them for 0.5 AGIX/ADA.
+Bob has 10 agix in his swap address and is willing to swap them for 1 ADA/AGIX.
+```
+
+In this example, Charlie can profitably arbitrage and fulfill both of these swaps like this:
+
+``` Txt
+Charlie looks up all swap addresses willing to swap AGIX/ADA. Charlie finds Alice's address.
+Charlie looks up all swap addresses willing to swap ADA/AGIX. Charlie finds Bob's address.
+Charlie gives Bob 10 ada and receives 10 agix.
+Charlie gives Alice 5 agix and receives 10 ada.
+Charlie now has his original 10 ada plus an additional 5 agix.
+This all occurs in one transaction where Charlie pays the transaction fee.
+```
 
-### Offer-Based Swap Addresses
-Currently, swap addresses are unique for every trading pair. This necessitates many distinct Beacon UTxOs (and associated 20 ADA deposits), even if users are offering the same asset in exchange for varying assets. Future versions of Cardano-Swaps may instead utilize "Offer-based" swap addresses, which are unique only to the asset being offered. The desired "exchange" asset would be specified in the datum of every "Swap" UTxO at the address. This would dramatically lower Beacon UTxO deposit fees, as only one such deposit is needed per "Offer" asset, per user. It would also make querying more expressive, allowing users to filter through available swaps not only by pairs, but by Offers too.
+On net, Charlie pays the transaction fees and receives 5 AGIX in return, while both Alice's and
```

Comment:
YMMV with other cryptocurrencies.  We'll probably encounter some other ones which insist that the cryptocurrency are units are capitalised, to match common usage based on obsession with crypto markets & trading.

We can't please everyone let alone match this divergence so I think the community would leave it up to you.  The people employing correct standards are often in the minority & I wouldn't want to place your material at a disadvantage for it: even if the matter is clear for `ada` vs. `ADA`.

For instance I followed the EU guidelines to call the new currency `euro` (not capitalised, always singular) since before the notes were even printed: then watched for 2 decades while everyone called the notes & values "euros" and/or Euro even in government communications. 🤪

Reply:
That sounds reasonable enough for me.  It's the Cardano Foundation that makes the repeated calls about under-capitalising the currency unit.  If & when it comes up in CIPs, other standards discussions & the community I'll do what I can do help justify the use of ADA if you choose... for the general reason that the platform itself is centred on trading.